openapi-cli/src/test/resources/generators/client/swagger/unsupported_request_body.yaml

```diff
@@ -48,7 +48,7 @@ paths:
           required: true
       requestBody:
         content:
-          multipart/form-data:
+          multipart/byteranges:
```

Comment:
Shall we mention with a comment stating this is a non-existing content type?

Reply:
yes,. Addressed in https://github.com/ballerina-platform/openapi-tools/commit/1eae7688b4b2792493641a66c733a3581b0f0ae4